lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -241,12 +245,18 @@
             );
             $( this.el ).html( renderedHtml );
 
+            var $resetBtn = this.$el.find('#webcam_reset_button'),
+                $captureBtn = this.$el.find('#webcam_capture_button');
+
```

Comment:
@aamir-khan while the variables cannot be assigned at the top, they can be declared at the top and then assigned here.